package/yast2-network.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Tue Feb  1 06:28:14 UTC 2022 - Michal Filka <mfilka@suse.com>
+
+- jsc#SLE-22015
+  - renamed CLI options to conform inclusive naming
```

Comment:
```suggestion
  - renamed CLI options to conform to inclusive naming:
    yast lan add name=bond0 bond_ports=eth0 eth1 bootproto=dhcp
```

Reply:
@mvidner Updated partly according to the proposal, partly using own creativity.